index.js

```diff
@@ -1,17 +1,27 @@
 'use strict';
 const ghGot = require('gh-got');
 
-module.exports = async (username, token) => {
+const defaultOptions = {
+	headers: {
+		'user-agent': 'https://github.com/sindresorhus/gh-user'
+	}
+};
+
+module.exports = async (username, options) => {
 	if (typeof username !== 'string' || !username) {
 		throw new Error('`username` required');
 	}
 
-	const result = await ghGot(`users/${username}`, {
-		token,
-		headers: {
-			'user-agent': 'https://github.com/sindresorhus/gh-user'
+	if (typeof options === 'object') {
+		options = {...defaultOptions, ...options};
```

Comment:
Spreading only work on one level, so the `header` option, for example, will not be correctly handled. You should use https://github.com/sindresorhus/got/blob/master/readme.md#gotmergeoptionsparentoptions-newoptions instead.